core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2685,3 +2685,52 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-aws-2
     namespace: ci
+- from:
+    .dockerconfigjson:
+      dockerconfigJSON:
+      - auth_bw_attachment: auth
+        bw_item: api.ci-pull-secret
+        registry_url: registry.svc.ci.openshift.org
+      - auth_bw_attachment: token_image-puller_app.ci_reg_auth_value.txt
+        bw_item: build_farm
+        registry_url: registry.ci.openshift.org
+      - auth_bw_attachment: auth
+        bw_item: cloud.openshift.com-pull-secret
+        email_bw_field: email
+        registry_url: cloud.openshift.com
+      - auth_bw_attachment: auth
+        bw_item: quay.io-pull-secret
+        email_bw_field: email
+        registry_url: quay.io
+      - auth_bw_attachment: auth
+        bw_item: registry.connect.redhat.com-pull-secret
+        email_bw_field: email
+        registry_url: registry.connect.redhat.com
+      - auth_bw_attachment: auth
+        bw_item: registry.redhat.io-pull-secret
+        email_bw_field: email
+        registry_url: registry.redhat.io
+  to:
+  - cluster: app.ci
+    name: pull-secret
```

Comment:
The content is exactly as `cluster-secrets-aws` but only the pull secret part.

https://github.com/openshift/release/blob/8554c3bd2bc84b99c4e4fb147448ded4ef4356b2/core-services/ci-secret-bootstrap/_config.yaml#L873-L876